src/products/views/ProductVariantCreate.tsx

```diff
@@ -133,7 +167,11 @@ export const ProductVariant: React.FC<ProductVariantCreateProps> = ({
         })}
       />
       <ProductVariantCreatePage
-        currencySymbol={shop?.defaultCurrency}
+        channels={channels}
+        channelErrors={
```

Comment:
Maybe we can set ProductVariantCreatePage to default channelErrors to `[]` so we don't have to care to always provide [] ?

Reply:
It's used only 2 times, so no real added value here I think